client/a8c-for-agencies/components/layout/index.tsx

```diff
@@ -24,6 +25,13 @@ export default function Layout( {
 	compact,
 	sidebarNavigation,
 }: Props ) {
+	const hasLayoutColumns = React.Children.toArray( children ).some(
+		( child ) => React.isValidElement( child ) && child.type === LayoutColumn
+	);
+	const layoutContainerClassname = ! hasLayoutColumns
+		? 'a4a-layout__container'
+		: 'a4a-layout-with-columns__container';
+
```

Comment:
I'm wondering if we could achieve the same behavior without this class conditional for wrapping the columns with `a4a-layout-with-columns__container` class. As an alternative, we could introduce a wrapper component `LayoutColumnsContainer`or `LayoutColumns`, like  the `LayoutNavigation` and `LayoutNavigationTabs`

Reply:
We could, but this seemed like a good simple solution that can be easily replaced by multiple layers of html blocks if we have this need in the future.